lstchain/scripts/lstchain_dvr_pixselector.py

```diff
@@ -168,19 +177,12 @@ def main():
         log.error('Unknown option in --action flag!')
         sys.exit(1)
 
-    dl1_files = []
-
-    if (write_pixel_masks or 
-        len(all_dl1_files) <= max_number_of_processed_subruns):
```

Comment:
this was a safe check, why removing it?

Reply:
This was to just process all input DL1 if the total number was smaller than the maximum we want to process (10)
But now the whole thing is done inside get_input_files

In case we are writing the pixel masks we do want all files to be processed, that is why the "if"